### PR TITLE
Release 0.64.1

### PR DESCRIPTION
**Pull request by rodrigok**
`  ·  1 🎉  ·  2 🐛  ·  3 🔍  ·  1 👩‍💻👨‍💻`

### 🎉 New features

- Store the last sent message to show bellow the room's name by default ([#10597](https://github.com/RocketChat/Rocket.Chat/pull/10597))

### 🐛 Bug fixes

- E-mails were hidden some information ([#10615](https://github.com/RocketChat/Rocket.Chat/pull/10615))
- Regression on 0.64.0 was freezing the application when posting some URLs ([#10627](https://github.com/RocketChat/Rocket.Chat/pull/10627))

<details>
<summary>🔍 Minor changes</summary>

- Support passing extra connection options to the Mongo driver ([#10529](https://github.com/RocketChat/Rocket.Chat/pull/10529) by [@saplla](https://github.com/saplla))
- Regression: Updating an App on multi-instance servers wasn't working ([#10611](https://github.com/RocketChat/Rocket.Chat/pull/10611))
- Dependencies update ([#10648](https://github.com/RocketChat/Rocket.Chat/pull/10648))

</details>

### 👩‍💻👨‍💻 Contributors 😍

- [@saplla](https://github.com/saplla)

### CODE REVIEW

File: tests/end-to-end/ui/04-main-elements-render.js [hunk 1]
```diff
@@ -137,7 +137,7 @@ describe('[Main Elements Render]', function() {
 				mainContent.favoriteStar.isVisible().should.be.true;
 			});
 
-			it('it should click the star', () => {
+			it.skip('it should click the star', () => {
```
Comment (sampaiodiego) [inline tests/end-to-end/ui/04-main-elements-render.js]: any reason for this being skipped?

Comment (rodrigok) [reply tests/end-to-end/ui/04-main-elements-render.js]: Yeap, it was failing duo to the last message been active, I wasn't able to discover why, so I disabled it

Comment (rodrigok) [reply tests/end-to-end/ui/04-main-elements-render.js]: @karlprieb can you take a look on it?

**Comment by neurolabs** [comment]
Thanks, ppl for pushing this release forward.

**Pull request merged**